src/herder/Upgrades.cpp

```diff
@@ -907,7 +910,8 @@ prepareLiabilities(AbstractLedgerTxn& ltx, LedgerTxnHeader const& header)
 
                 // the deltas should only be positive when liabilities were
                 // introduced in ledgerVersion 10
-                if (header.current().ledgerVersion > 10 &&
+                if (protocolVersionStartsFrom(header.current().ledgerVersion,
```

Comment:
This seemingly uncovers a bug (or at least inconsistency between comment and code), but I wanted this change to be no-op.